packages/blade/src/components/Box/Box.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import BaseBox from './BaseBox';
+import type { BoxProps, MakeValueResponsive } from './BaseBox/types';
+import type { KeysRequired } from '~src/_helpers/types';
+import { metaAttribute, MetaConstants } from '~utils';
+
+const validateBackgroundString = (stringBackgroundColorValue: string): void => {
+  if (!stringBackgroundColorValue.startsWith('surface.background')) {
+    throw new Error(
+      `[Blade - Box]: Oops! Currently you can only use \`surface.background.*\` tokens with backgroundColor property but we received \`${stringBackgroundColorValue}\` instead.\n\n Do you have a usecase of using other values? Create an issue on https://github.com/razorpay/blade repo to let us know and we can discuss ✨`,
+    );
+  }
+};
+
+const validateBackgroundProp = (
+  responsiveBackgroundColor: MakeValueResponsive<string | undefined>,
+): void => {
+  if (responsiveBackgroundColor) {
+    if (typeof responsiveBackgroundColor === 'string') {
```

Comment:
didn't you remove this code as you had mentioned in the comments?

Reply:
Oh it is removed from that forEach block. We still need it here. We say if it is "string" then handle it here itself or else do that `.forEach` thing and handle it there. So it can be string or an object.

<img width="803" alt="image" src="https://user-images.githubusercontent.com/30949385/225217639-671021bd-2aae-46c0-913b-cea181be48ab.png">